internal/ledger/artifact/artifact.go

```diff
@@ -160,35 +159,17 @@ func (m *Scope) activateObject(
 	parent insolar.Reference,
 	memory []byte,
 ) error {
-	parentIdx, err := m.IndexAccessor.ForID(ctx, m.PulseNumber, *parent.Record())
```

Comment:
we kept parent check in artifacts.client... to make sure parent reference is valid and not bogus

Reply:
Genesis doesn't have any checks. It just writes data into storage from config.

Reply:
this if below makes sure that the parent exists. what is wrong with keeping these four lines here?